docs/eep.schema

```diff
@@ -94,8 +94,8 @@
                                 "gpio": {
                                     "description": "GPIO or pin number",
                                     "type": "integer",
-                                    "minimum": 2,
-                                    "maximum": 27
+                                    "minimum": 0,
+                                    "maximum": 45
```

Comment:
I would prefer to add changes to schema when the implementation is done.

Reply:
This will be done here: https://github.com/RevolutionPi/revpi-hat-eeprom/pull/21

Reply:
That other pull request came in later and lacks a proper explanation for the change. ;)